packages/core/contracts/financial-templates/perpetual-multiparty/PerpetualPositionManager.sol

```diff
@@ -128,7 +128,7 @@ contract PerpetualPositionManager is FundingRateApplier {
      */
     constructor(
         uint256 _withdrawalLiveness,
-        address _collateralAddress,
+        address _collateralTokenAddress,
         address _tokenAddress,
```

Comment:
we need a consistent way to name the collateral vs the synthetic. I think previously any time "token" is mentioned its in the context of the synthetic, while collateral is just collateral. With this change we add collateralTokenAddress which  makes sense on its face, but its followed by tokenAddress, rather than something like syntheticTokenAddress.  I think im actually ok with just collateralAddres/tokenAddress, but what do you all think?

Reply:
it seems like most of the changes have gone with synthetic, which is fine, then probably tokenAddress should change to syntheticTokenAddress

Reply:
I think you're totally right about being consistent and that we have historically referenced `collateral` and `token`.

I actually agree we should keep this `_collateralAddress` and keep `_tokenAddress` as is because we are not renaming the public variable `tokenCurrency`. We shouldn't have both `tokenCurrency` and `_syntheticAddress`.

In summary: it sounds like we should leave `_collateralAddress` as is.

Reply:
+1